songbird/queue.py

```diff
@@ -87,23 +88,23 @@ def append(self, __object: T) -> None:
         self.item_added.set()
         return super().append(__object)
 
-    def extend(self, __iterable: Iterable[T]) -> None:
+    def extend(self, __iterable: typing.Iterable[T]) -> None:
         self.item_added.set()
         return super().extend(__iterable)
 
-    def insert(self, __index: SupportsIndex, __object: T) -> None:
+    def insert(self, __index: typing.SupportsIndex, __object: T) -> None:
         self.item_added.set()
         return super().insert(__index, __object)
 
     def __add__(self, __x: list[T]) -> list[T]:
         self.item_added.set()
         return super().__add__(__x)
 
-    def __iadd__(self: Queue, __x: Iterable[T]) -> Queue:
+    def __iadd__(self: Queue, __x: typing.Iterable[T]) -> Queue:
         self.item_added.set()
         return super().__iadd__(__x)
 
-    async def _play_next(self, *args) -> None:
+    async def _play_next(self, *_args: typing.Any) -> None:
```

Comment:
theres not a reason to make this protected

Reply:
Its google style, this practice is used a lot where the variable is not used in the code